validator/lib/readTileset.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
I didn't put this in the issue, but can you write tests for this function? Look at `readTileSpec` for an example. A couple things to check:
* a normal JSON file
    * Use `specs/data/Tileset/tileset.json` as your test case
* a gzipped JSON file
    * We already have a tileset at `specs/data/Tileset`. Create a gzipped tileset at `specs/data/TilesetGzipped`. You can use the gzip utility in https://github.com/AnalyticalGraphicsInc/3d-tiles-tools/tree/master/tools to generate a gzipped tileset from `specs/data/Tileset`. Then use `specs/data/TilesetGzipped/tileset.json` as your test case.
* a file that isn't valid JSON
    * look at https://github.com/AnalyticalGraphicsInc/3d-tiles-tools/blob/master/tools/specs/lib/gzipTilesetSpec.js#L155 for an example of checking a rejected promise
    * As a test file you can use one of the b3dm's in the `specs/data/Tileset` folder